src/refer.ts

```diff
@@ -81,32 +100,44 @@ function refer(this: any, options: any) {
     }
   }
 
-
   async function prepare(this: any) {
-    let seneca = this
+    const seneca = this
     await seneca.post('biz:refer,load:rules')
   }
-
 }
 
+async function getEntryReward(seneca: any, entry: any): Promise<object> {
+  const reward = await seneca.entity('refer/reward').load$({
+    entry_id: entry.id
+  })
+
+  if (!reward) {
+    return await seneca.entity('refer/reward').save$({
+      entry_kind: entry.kind,
+      entry_id: entry.id,
+      kind: 'accept',
+      count: 1
+    })
+  }
+
+  reward.count = reward.count + 1
+  return await seneca.entity('refer/reward').save$(reward)
```

Comment:
`reward.save$()` is better
see https://senecajs.org/docs/tutorials/understanding-data-entities.html

Reply:
Indeed! I followed your suggestion, please take a look again.